python_frank_energie/models.py

```
@@ -1,16 +1,28 @@
+# python_frank_energie/models.py
 
 import logging
-from dataclasses import dataclass
-from datetime import datetime, timedelta, timezone
-
+import statistics
+from collections import defaultdict, namedtuple
+from dataclasses import dataclass, field
+from datetime import date, datetime, timedelta, timezone
+from statistics import mean
+from pydantic import BaseModel, Field, EmailStr
```

Ruff

`pydantic.Field` imported but unused

Remove unused import: `pydantic.Field`

(F401)